src/styles/catalog.scss

```diff
@@ -0,0 +1,9 @@
+.catalog {
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  row-gap: 46px;
+  padding: 50px 40px;
+
+  @include on-all;
```

Comment:
The '@include on-all;' directive is used here, but it's not clear if 'on-all' is defined elsewhere in your SCSS files. Ensure that 'on-all' is a valid mixin or remove this line if it's not needed.